src/app/views/sidebar/history/History.tsx

```diff
@@ -364,19 +373,22 @@ export class History extends Component<IHistoryProps, any> {
       actions.setSampleQuery(sampleQuery);
       actions.runQuery(sampleQuery);
     }
+    this.trackHistoryItemEvent(BUTTON_CLICK_EVENT, 'Run History Item Button', query);
   }
 
   private onExportQuery = (query: IHistoryItem) => {
     const harPayload = createHarPayload(query);
-    const generatedHarData = generateHar([harPayload]);
+    const generatedHarData = generateHar([harPayload]);   
     exportQuery(generatedHarData, `${query.url}/`);
+    this.trackHistoryItemEvent(BUTTON_CLICK_EVENT, 'Export History Item Button', query);
   }
 
   private deleteQuery = async (query: IHistoryItem) => {
-    const { actions } = this.props;
+    const { actions } = this.props;     
     if (actions) {
-      actions.removeHistoryItem(query);
+        actions.removeHistoryItem(query);
     }
+    this.trackHistoryItemEvent(BUTTON_CLICK_EVENT, 'Delete History Item Button', query);
```

Comment:
are these names easy to query on?

Reply:
Yes, I'll share the queries on the design doc.